SECURITY.md

```diff
@@ -0,0 +1,3 @@
+# Caliptra Project Security Incident Response
+
+Please refer to the security policy at [Caliptra security policy](https://github.com/chipsalliance/caliptra/security/policy).
```

Comment:
Actually: isn't this Caliptra/security.md

Reply:
The link above is correct. The page will contain the security policy and other useful links once SECURITY.md is merged into the main Caliptra repo.

Reply:
Here's what the equivalent page looks like in my sample repo: https://github.com/FerralCoder/Test/security/policy